.storybook/webpack.config.js

```diff
@@ -20,28 +20,28 @@ function getBuildOuptutPathList(buildOutputRelativePath) {
 
 // Mock ROOT_GEN_DIR as 'gen' - it will be replaced with an array from
 // getBuildOutputPathList.
-process.env.ROOT_GEN_DIR = 'gen'
+const potentialGenFolders = getBuildOuptutPathList('gen')
+process.env.ROOT_GEN_DIR = potentialGenFolders
+  .filter(a => fs.existsSync(a))
+  .sort((a, b) => fs.statSync(a).mtime - fs.statSync(b).mtime)[0]
+
+  // Fallback to the first folder in the list, so we get sensible errors.
+  ?? potentialGenFolders[0]
+
 const basePathMap = require('../components/webpack/path-map')
```

Comment:
probably a less hacky way than making sure the env variable is defined before the first `require` of `path-map.js` is to have path-map return a function that takes a prefix and have both webpacks pass in different variables. But at least it works for now.